httpie/core.py

```
@@ -0,0 +1,202 @@
+    Convert all bytes ags to str
+    by decoding them using stdin encoding.
+
+    """
+    return [
+        arg.decode(stdin_encoding)
+        if type(arg) == bytes else arg
+        for arg in args
+    ]
+
+
+def main(args=sys.argv[1:], env=Environment(), error=None):
```

Ruff (0.8.2)

Do not perform function call `Environment` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)